std/random.d

```diff
@@ -1602,7 +1602,7 @@ and initialized to an unpredictable value for each thread.
 Returns:
 A singleton instance of the default random number generator
  */
-@property ref Random rndGen() @safe @nogc
+@property ref Random rndGen() @safe nothrow @nogc
```

Comment:
Please also add `nothow` and `@nogc` to the unittest below.

Reply:
Done.